src/mono/mono/metadata/class-init.c

```diff
@@ -3394,7 +3394,8 @@ mono_class_setup_methods (MonoClass *klass)
 			if (methods [i]->flags & METHOD_ATTRIBUTE_VIRTUAL)
 			{
 				if (method_is_reabstracted (methods[i]->flags)) {
-					methods [i]->is_reabstracted = 1;
+					if (!methods [i]->is_inflated)
```

Comment:
This is a behavioral change, right? What's the intended outcome if we end up here with an inflated method?

Reply:
this is a property of the method _definition_ which means for a generic instance we should look at the declaring method (which is what the lookup methods for the infrequent bits do)